types/prismic-dom/index.d.ts

```diff
@@ -9,7 +9,12 @@ interface RichText {
     asText(richText: any,  joinString?: string): string;
 }
 
+interface Link {
+    url(link: any, linkResolver?: (doc: any) => string): string;
+}
+
 export const RichText: RichText;
+export const Link: Link;
 
 declare const _default: { RichText: RichText };
```

Comment:
I think you'll need to also add `Link` to the default exports too.

https://github.com/prismicio/prismic-dom/blob/master/src/index.js#L4

Reply:
Should be fixed now!